Invalid Token Error Solution (#5173)
segmentio/segment-docs: src/connections/destinations/catalog/actions-snap-conversions/index.md

## FAQ and Troubleshooting

> ### Invalid Token Error

Reviewer: ```suggestion
### Invalid token error
```

> If you are experiencing many 400 Bad Requests errors related to an invalid token, please make sure you reauthorize your account by following the steps [here](https://segment.com/docs/connections/destinations/catalog/actions-snap-conversions/#getting-started): On the Settings tab, authenticate with Snap using OAuth. Click Connect to Snapchat Conversions API. Follow the prompts to authenticate using OAuth, with a Snapchat login that is a member of the Snapchat Ads account you want to connect.

Reviewer: ```suggestion
If you're experiencing 400 Bad Requests errors related to an invalid token, follow [these instructions](https://segment.com/docs/connections/destinations/catalog/actions-snap-conversions/#getting-started) to reauthorize your account: 
- On the **Settings** tab, authenticate with Snap using OAuth. 
- Click **Connect to Snapchat Conversions API**. 
- Follow the prompts to authenticate using OAuth with a Snapchat login. Use a Snapchat login that is a member of the Snapchat Ads account you want to connect.
```
